commands/apps/index.js

```diff
@@ -88,12 +89,12 @@ Example:
  === Collaborated Apps
  theirapp   other@owner.name`,
   needsAuth: true,
-  wantsOrg: true,
   flags: [
     {name: 'all', char: 'A', description: 'include apps in all organizations'},
     {name: 'json', description: 'output in json format'},
     {name: 'space', char: 's', hasValue: true, description: 'filter by space'},
-    {name: 'personal', char: 'p', description: 'list apps in personal account when a default org is set'}
+    {name: 'personal', char: 'p', description: 'list apps in personal account when a default org is set'},
+    flags.team({name: 'team', hasValue: true})
```

Comment:
we still need the org flag I think but make it hidden. Also, does this respect the `HEROKU_ORGANIZATION` env var? we should also add `HEROKU_TEAM` if we don't have it already now that I think about it